### DISCUSSION TITLE
The Dark Side of .io

### DISCUSSION THREAD

**Discussion by janxkoci** [elementary/website#3108] [General]
### Problem

Since you are using the .io domain for your main site (as many FOSS projects do), the issues that come with this domain relate to you too. In short, this TLD belongs to the Chagos Archipelago, which suffered a mass deportation by the UK government 50 years ago. These people still fight for the right to return to their homeland.

There is a campaign to support this cause here: http://www.thedarksideof.io/

### Proposal

There are a few options you could consider:

1. You could make a page informing about the issue and show the donate button, or at least link to the site mentioned above.
2. You could move your site to another TLD (as some [other projects](https://privacyguides.org/blog/2021/09/14/welcome-to-privacy-guides/) did).
3. You could of course ignore the issue and keep doing what you're doing. This is more of an FYI, really.

I think option 1. is the best, but I will leave the decision to you.

**Comment by kittylq** [comment]
The web site darksideof.io doesn't show anything any more. It seems like if it's not "trendy" like Palestine, suddenly people just don't care about justice any more. It would be good to bring more visibility to this
